modules/metarefresh/lib/MetaLoader.php

```diff
@@ -124,6 +124,29 @@ public function loadSource($source) {
 			return;
 		}
 
+        $filterFunction = null;
+        if (isset($source['filterCallback'])) {
+            if (!is_callable($source['filterCallback'])) {
+                SimpleSAML_Logger::debug('Invalid filter callback ' . $source['filterCallback'] . ' - attempting to re-use cached metadata');
```

Comment:
I'm not sure this should be `debug`. In the end, this would be a configuration error, right? So in that case you want the user to notice about it, even though in most deployments people won't use *debug* as log level.